hydra-web/src/js/app.js

```diff
@@ -292,6 +352,19 @@ function(
                 $("span#quiescedLabel").hide();
                 $("#topNavbar").removeClass("navbar-inverse");
             }
+        },
+        sudoCb:function(checked, show){
```

Comment:
I assume `sudoCb` stands for `sudoCheckbox`? Use a fuller name such as `showSudoCheckbox`